docs/src/modules/java/pages/writing-grpc-descriptors-protobuf.adoc

```diff
@@ -62,48 +63,33 @@ This section of the `.proto` file declares the API of the service itself, along
 
 The `AddItem` and `RemoveItem` methods have no return value (the `Empty` type).
 
-See <<_transcoding_http>> for an explanation of the HTTP annotations.
+Most important, in this file we instruct the Akka Serverless Codegen which kind of component we want to create. The codegen will generated all stubs for your entity/service and corresponding tests, as well as an abstract class for your implementation to extend.
```

Comment:
```suggestion
Most important, in this file we instruct the Akka Serverless Codegen which kind of component we want to create. The codegen will generate all stubs for your entity/service and corresponding tests, as well as an abstract class for your implementation to extend.
```

Reply:
My inclination would be to use `Most importantly` (sentence adverb rather than adjective) but that's not important.